shared/js/background/dnr-click-to-load.js

```diff
@@ -69,6 +69,51 @@ async function generateDnrAllowingRules (tab, ruleAction) {
     return allowingRules
 }
 
+/**
+ * Find the enabled Click to Load rule actions for the given tab.
+ * Note: Take care to ensure wait for the extension configuration to be ready
+ *       first.
+ * @param {import('./classes/tab.es6')} tab
+ * @return {string[]}
+ */
+export function getDefaultEnabledClickToLoadRuleActionsForTab (tab) {
```

Comment:
This is not DNR related right? Why is it in the `dnr-click-to-load` file? Should we maybe just rename the whole file to `click-to-load` if it is manifest-version agnostic?

Reply:
Mind if I leave that for now? I thought the same, but if we rename the file some of the other exported function names are less obviously relating to the declarativeNetRequest API. I should probably rename those too, but this PR is already getting huge, and equally having a module just for this function seems like overkill.